packages/sanes-chrome-extension/src/logic/persona/persona.ts

```diff
@@ -12,6 +21,19 @@ import {
 } from '../config';
 import { AccountManager, AccountInfo, AccountManagerChainConfig } from './accountManager';
 
+/** Like JsRpcSigningServer but without functionality to create or shutdown */
+export interface UseOnlyJsRpcSigningServer {
```

Comment:
I prefer to move this away from Persona.

Reply:
See discussion above